port_name            : "/dev/urbtc1"
ros_rate             : 100
wheel_base           : 0.94
tread                : 0.43515
wheel_diameter_right : 0.38725
wheel_diameter_left  : 0.38695
max_linear_vel       : 1.1
max_angular_vel      : 3.141519
gain_p_right         : 1.0
gain_i_right         : 0
gain_d_right         : 0
gain_p_left          : 1.0
gain_i_left          : 0
gain_d_left          : 0
motor_pin_right      : 103
motor_pin_left       : 102
enc_pin_right        : 107
enc_pin_left         : 106
motor_pin_offset     : 101
enc_pin_offset       : 105
motor_pin_ch_right   : 1
motor_pin_ch_left    : 2
rc_filter_rate       : 0.9
